buildSrc/build.gradle

```diff
@@ -0,0 +1,11 @@
+plugins {
```

Comment:
Do we need these settings in a `buildSrc` for some specific reason? I think we can simply have a top-level project with some modules using OOTB gradle settings. This is an example that's much simple to wire: https://github.com/cosmin-marginean/slink/blob/main/build.gradle
Happy to do this multi-module setup instead if you want me to.

Reply:
Hi @cosmin-marginean, please review the multi-module changes in #4 instead. I will rebase this onto `main` after #4 is merged

Reply:
Thanks, that's good to know! I was behind the latest with Gradle to be honest.